lms/djangoapps/api_manager/users/views.py

```diff
@@ -916,6 +916,35 @@ def delete(self, request, user_id, course_id):
         return Response({}, status=status.HTTP_204_NO_CONTENT)
 
 
+class UsersCoursesGradesList(SecureAPIView):
+    """
+    ### The UsersCoursesGradesList view allows clients to interact with the User's gradebook across courses
+    - URI: ```/api/users/{user_id}/courses/grades?courses={course_id1,course_id2}```
+    - GET: Returns a JSON representation of the matched set from the gradebook
+        * courses (optional): Set filtering parameter consisting of comma-separated course identifiers
+
+    ### Use Cases/Notes:
+    * Use the UsersCoursesGradesList view to interact with the User's gradebook across multiple Course enrollments
+    * Use GET to retrieve all of the Course gradebook entries for the specified User
+    """
+    def get(self, request, user_id):  # pylint: disable=unused-argument
```

Comment:
The 'request' parameter is a Django Rest Framework necessity -- you can see examples in other API operations, too.  We can utilize it in the method but even if we don't we need to include it.  I'm not sure why Pylint is complaining about it now (and thus the need for the disable flag).